x-pack/plugins/cross_cluster_replication/public/app/sections/home/follower_indices_list/components/follower_indices_table/follower_indices_table.js

```diff
@@ -64,6 +69,100 @@ export const FollowerIndicesTable = injectI18n(
     getTableColumns() {
       const { intl, selectFollowerIndex } = this.props;
 
+      const actions = [
+        /* Pause or resume follower index */
+        {
+          render: ({ name, shards }) => {
+            const isPaused = !shards || !shards.length;
```

Comment:
Maybe this status is something we should calculate during deserialization? That way our logic for defining the index status lives in one place.

Reply:
this will be removed once the new list api is integrated! (that will give us a separate `status` field natively)